tests/components/onewire/devices/test_family_00.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for 1-Wire device family 00 (Invalid device)."""
+from os import path
+from unittest.mock import mock_open, patch
+
+from homeassistant.components.onewire.const import DEFAULT_OWSERVER_PORT
+import homeassistant.components.sensor as sensor
+from homeassistant.setup import async_setup_component
+
+from tests.common import mock_registry
+
+OWFS_MOUNT_DIR = "/mnt/OneWireTest"
+
+DEVICE_ID = "00.111111111111"
+DEVICE_NAME = "My invalid device"
+
+
+async def test_setup_owfs(hass):
+    """Test a device which is not recognised."""
+    entity_registry = mock_registry(hass)
+    config = {
+        "sensor": {
+            "platform": "onewire",
+            "mount_dir": OWFS_MOUNT_DIR,
+            "names": {
+                DEVICE_ID: DEVICE_NAME,
+            },
+        }
+    }
+    mo_main = mo_family = mock_open(read_data=DEVICE_ID[0:2])
```

Comment:
We don't want to add tests of the one wire integration parts that doesn't yet use a 3rd part library for its device communication.

Reply:
I've removed the tests in 2c1690f, and added corresponding "pargma: no cover" in c858d44 with extra comments.